x-pack/plugins/security_solution_serverless/server/constants.ts

```diff
@@ -9,4 +9,4 @@
 const namespace = 'elastic-system';
 const USAGE_SERVICE_BASE_API_URL = `http://usage-api.${namespace}/api`;
 const USAGE_SERVICE_BASE_API_URL_V1 = `${USAGE_SERVICE_BASE_API_URL}/v1`;
-export const USAGE_SERVICE_USAGE_URL = `${USAGE_SERVICE_BASE_API_URL_V1}/usage`;
+export const USAGE_SERVICE_USAGE_URL = `${USAGE_SERVICE_BASE_API_URL_V1}`;
```

Comment:
is this on purpose? I believe we need the `/usage`: https://github.com/elastic/usage-api/blob/5414a1c5f170b81552deb2140307e135ad93c7e0/api/user-v1-spec.yml#L50

Reply:
you right:)